app/js/components/breadCrumbs/breadCrumb.js

```diff
@@ -23,12 +40,12 @@ class BreadCrumbComponent extends React.Component {
           </Breadcrumb>
           : (location.pathname.includes('/patient/') && location.pathname.includes('/visit/')) ?
             <Breadcrumb>
-              <BreadcrumbItem id="visited" onClick={router.goBack}> <a id="home"
+              <BreadcrumbItem data-id="search" onClick={this.handleRoute.bind(this)}> <a id="home"
                 href="https://modules-refapp.openmrs.org/openmrs/index.htm">
                 <FontAwesome className='fa fa-home' name='home' /></a>
                 < FontAwesome name='chevron-right' className='fa fa-chevron-right' />
                  Search</BreadcrumbItem>
-              <BreadcrumbItem id="visited" onClick={router.goBack}>
+              <BreadcrumbItem data-id="patient_visit" onClick={this.handleRoute.bind(this)}>
                 <FontAwesome name='chevron-right' className='fa fa-chevron-right' />
                 Patient Data</BreadcrumbItem>
               <BreadcrumbItem active>
```

Comment:
Fix the indentation levels to make your code more readable

Reply:
This has been rectified.Thanks